tests/test_webapp.py

```diff
@@ -0,0 +1,91 @@
+#
+# test_webapp.py
+#
+# Copyright (C) 2017 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Tests for the WebApp browser
+#
+
+
+import os
+import mock
+
+
+def test_navigation_requests(webapp, navigation_action):
```

Comment:
As a note for the future, I find it easier to test specific functions. In this case this tests `_nav_req_handler`, `_api_handler`, and `_scheme_handler`. When you scope down your tests you'll find that you'll only ever need to do one `assert` at the end as well.